config/pure-python-with-multiprocess/tox.ini

```diff
@@ -0,0 +1,61 @@
+# Generated from:
+# https://github.com/zopefoundation/meta/tree/master/config/pure-python-with-multiprocess
+[tox]
+envlist =
+    lint,
+    py27,
+    pypy,
+    py35,
+    py36,
+    py37,
+    py38,
+    pypy3,
+    coverage
+
+[testenv]
+usedevelop = true
+deps =
+    zope.testrunner
+commands =
+    zope-testrunner --test-path=src []
+extras = test
+
+[testenv:lint]
+basepython = python3
+skip_install = true
+deps =
+    flake8
+    check-manifest
+    check-python-versions
+commands =
+    flake8 src setup.py
+    check-manifest
+    check-python-versions
+
+[testenv:coverage]
+basepython = python3
+setenv =
+    COVERAGE_PROCESS_START={{toxinidir}}/.coveragerc
+deps =
+    coverage
+    coverage-python-version
+    zope.testrunner
+commands =
+    coverage erase
+    coverage run -m zope.testrunner --test-path=src []
+    coverage combine
+    coverage html
+    coverage report -m {coverage_report_options}
+
+[coverage:report]
```

Comment:
This should be in .coveragerc.

Reply:
I've overlooked this section, moved it now to `.coveragerc`.